ketcher-autotests/tests/User-Interface/Left-toolbar/left-toolbar-ui.spec.ts

```diff
@@ -0,0 +1,115 @@
+import { test, expect } from '@playwright/test';
+import {
+  BondType,
+  clickOnBond,
+  drawBenzeneRing,
+  takeLeftToolbarScreenshot,
+} from '@utils';
+
+test.describe('Left toolbar UI tests', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('');
+  });
+
+  test('left toolbar ui verification', async ({ page }) => {
+    // Test case: EPMLSOPKET-4268
+    await takeLeftToolbarScreenshot(page);
+  });
+
+  test('left toolbar selection tool verification', async ({ page }) => {
+    // Test case: EPMLSOPKET-4268
+    await page.getByTestId('select-rectangle').click();
+    await page.getByTestId('select-rectangle').click();
+    const bodyHeight = await page.evaluate(() => document.body.clientHeight);
```

Comment:
Please, create a function for repetitive actions. Example:
```
// but actually it is better to calculate width of dropdown automatically in the function
// instead of passing number
async function takeDropdownScreenshot(page: Page, width: number) {
  const bodyHeight = await page.evaluate(() => document.body.clientHeight);
  const screenshot = await page.screenshot({
      clip: {
        x: 0,
        y: 0,
        width,
        height: bodyHeight,
      },
    });
    expect(screenshot).toMatchSnapshot();
}
```

And I propose you to consider https://playwright.dev/docs/test-parameterize

how your tests can look:

```
const dropdowns = [{
  tool: 'bonds',
  // better to use testId instead
  width: 700,
}, {
  tool: 'select-rectangle',
  width: 200,
}, 
// add other tools
]
for (const dropdown of dropdowns) {
  test(`Test ${dropdown.tool} dropdown`, () => {
    await page.getByTestId(dropdown.tool).click();
    await page.getByTestId(dropdown.tool).click();
    const bodyHeight = await page.evaluate(() => document.body.clientHeight);
    const screenshot = await page.screenshot({
      clip: {
        x: 0,
        y: 0,
        width: dropdown.width,
        height: bodyHeight,
      },
    });
    expect(screenshot).toMatchSnapshot();
  })
}
```

Reply:
fixed